packages/runtime/test/integration/execute-transaction.ts

```diff
@@ -113,17 +113,19 @@ describe("Algorand Smart Contracts - Execute transaction", function () {
     syncAccounts();
     const assetInfo = runtime.getAssetInfoFromName('gold');
     assert.isDefined(assetInfo);
-    const tx: types.ExecParams[] = [
-      {
-        type: types.TransactionType.OptInASA,
-        sign: types.SignType.SecretKey,
-        fromAccount: alice.account,
-        assetID: assetInfo?.assetIndex as number,
-        payFlags: { totalFee: 1000 }
-      }
-    ];
-
-    runtime.executeTx(tx);
+    if (assetInfo !== undefined) {
```

Comment:
this smells like a bug - why do we need an if here? If we really need it, then let's define `tx` outside:
```
let tx;
```
and overwrite it inside the if.
Also, let's document (add a comment, and in the test case below as well) why / when the assetInfo is undefined.

Reply:
this is actually extending the same issue with eslint. I am still checking how it's happening. But usually something like this is after the merge.